release_notes/unreleased.md

```diff
@@ -1 +1,7 @@
 **Unreleased**
+* Added the following actions [PAPP-9260, PAPP-9261, PAPP-21056, PAPP-21057]
+  * add host
+  * delete host
+  * list hosts
+  * logout session
```

Comment:
I'm guessing this was from a prior PR and the changes haven't been released yet?

Reply:
correct